Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,11 @@
 function sum(elements) {
 }
 
+module.exports = sum;
+ // Filter for numerical elements and sum them
+ return elements
+ .filter((element) => typeof element === "number")
+ .reduce((acc, num) => acc + num, 0); // Start with 0 as the initial sum
+}
+
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function return the value you expected?

```
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```